File: Basics/Conditional-Statements-Advanced/Lab/fruit_or_vegetable.py
product = input().lower()

fruit = ["banana", "apple", "kiwi", "cherry", "lemon", "grapes"]
vegetable = ["tomato", "cucumber", "pepper", "carrot"]

if product in fruit:
    print(f"fruit")
elif product in vegetable:
    print(f"vegetable")
else:
    print(f"unknown")

File: JavaScript/Basics/0.3-Conditional-Steatements-Advanced/Lab/fruit-or-vegetable.js
function fruitOrVegetable(input) {
    let food = input[0];
    let fruits = ["banana", "apple", "kiwi", "cherry", "lemon", "grapes"];
    let vegetables = ["tomato", "cucumber", "pepper", "carrot"];
    if (fruits.includes(food)) {
        console.log("fruit");
    } else if (vegetables.includes(food)) {
        console.log("vegetable");
    } else {
        console.log("unknown");
    }
}